test/udev-test.pl

```diff
@@ -1550,10 +1551,9 @@ sub run_test {
 print "$error errors occurred\n\n";
 
 # cleanup
-system("rm", "-rf", "$udev_dev");
 system("rm", "-rf", "$udev_run");
 system("umount", "$udev_tmpfs");
```

Comment:
> udev_setup() now does the umount right at the top

We need the `umount` here.
There is a `clean` flag: https://github.com/evverx/systemd/blob/udev-test-dont-rely-on-underlying-fs-2/test/udev-test.pl#L1492
```
        if (defined($rules->{option}) && $rules->{option} eq "clean") {
                udev_setup();
        }
```
Some tests rely on that flag.
There are three `mount/umount`:
```
ACTION     TARGET                          SOURCE FSTYPE OPTIONS
mount      /home/ubuntu/systemd/test/tmpfs tmpfs  tmpfs  rw,nosuid,nodev,noexec,relatime,mode=755
ACTION     TARGET                          SOURCE FSTYPE OPTIONS
umount     /home/ubuntu/systemd/test/tmpfs tmpfs  tmpfs  rw,nosuid,nodev,noexec,relatime,mode=755
ACTION     TARGET                          SOURCE FSTYPE OPTIONS
mount      /home/ubuntu/systemd/test/tmpfs tmpfs  tmpfs  rw,nosuid,nodev,noexec,relatime,mode=755
ACTION     TARGET                          SOURCE FSTYPE OPTIONS
umount     /home/ubuntu/systemd/test/tmpfs tmpfs  tmpfs  rw,nosuid,nodev,noexec,relatime,mode=755
ACTION     TARGET                          SOURCE FSTYPE OPTIONS
mount      /home/ubuntu/systemd/test/tmpfs tmpfs  tmpfs  rw,nosuid,nodev,noexec,relatime,mode=755
ACTION     TARGET                          SOURCE FSTYPE OPTIONS
umount     /home/ubuntu/systemd/test/tmpfs tmpfs  tmpfs  rw,nosuid,nodev,noexec,relatime,mode=755
```